index.html

```diff
@@ -1127,6 +1163,150 @@ <h2 property="schema:name">Discovering a complete Solid profile</h2>
+                                    rel="spec:requirementSubject" href="#ReaderApplication">Reader Application</a> <span
+                                    rel="spec:requirementLevel" resource="spec:MUST">MUST</span> MUST make
+                                an <code>HTTP GET</code> request targeting a Solid WebID Profile resource including a
+                                <code>Content-Type header</code> requesting a representation in <code>text/turtle</code> or
+                                <code>application/ld+json.</code></span>
+                        </p>
+                        <section id="webid-profile-data-model" inlist="" rel="schema:hasPart"
+                            resource="#webid-profile-data-model">
+                            <h3 property="schema:name" content="WebID Profile Data Model">Solid WebID Profile</h3>
+                            <div datatype="rdf:HTML" property="schema:description">
+                                <p>The <a href="#solid-profile">Solid WebID Profile</a> has the following properties:</p>
+                                <ul>
+                                    <li>One <code>rdf:type</code> property whose object is <code>foaf:Agent</code>.</li>
+                                    <li>One <code>space:preferencesFile</code>.</li>
+                                    <li>Zero or more <code>ldp:inbox</code>.</li>
+                                    <li>Zero, one, or more <code>space:storage</code>.</li>
+                                    <li>Zero, one, or more <code>rdfs:seeAlso</code> to refer to related <a
+                                            href="#extended-profile-documents">documents extending the profile.</a></li>
+                                </ul>
+                                <div class="note" id="other-information" inlist="" rel="schema:hasPart"
+                                    resource="#other-information">
+                                    <h5 property="schema:name"><span>Note</span>:</h5>
+                                    <div datatype="rdf:HTML" property="schema:description">
+                                        <p>The Solid WebID Profiles may include other information pertaining to scenarios in
+                                            which profiles are used or shared as mentioned in <a href="#use-cases">use
+                                                cases</a>.</p>
+                                    </div>
+                                </div>
+                            </div>
+                        </section>
+                    </div>
+        
+                    <h3 property="schema:name" content="Preferences Document">Preferences Document</h3>
+                    <div datatype="rdf:HTML" property="schema:description">
+                        <p>The <a href="#private-profile">Preferences Document</a> has the following properties:</p>
```

Comment:
Was `#preferences-document` or `#private-preferences` instead of `#private-profile` intended?

Reply:
Good question. I don't think that originated with me, so this should probably be asked outside of my review.